### ISSUE TITLE
Add link from Teams panel to github user repos

### ISSUE THREAD

**Issue by rtholmes** [ubccpsc/classportal#7]
The 'repos' button from #3 was super awesome. It would be great if the students could check if their team mates had registered as well. In the Teams panel for the student view, could the button change to the same style as for the 'repos' button if the teammate has their github username registered? Tapping on the button should take them to the github user page. Since the student checking will already be registered their own button will always show as a button that they can tap to go to their page too.

It would be good to extend #3 by making the repo button look disabled if the link hasn't been set yet.

**Comment by mksarge** [comment]
I'll look into this over the weekend. Perhaps something like:

![displayteam](https://cloud.githubusercontent.com/assets/19560975/18601255/b1ca73e0-7c16-11e6-89c0-d9c51543ec43.png)

**Comment by rtholmes** [comment]
I'm assuming the Team3 / Teammate1 / Teammate 2 things are all just depressed buttons right? I think you could just have those three buttons, and if they exist (aka the repo exists for the team, or the users are registered in github) they turn to actual pressable buttons, otherwise they are just in the depressed state. That way you don't actually need the repo ones beside?

**Comment by mksarge** [comment]
Yes, they're unselectable/unclickable forms. I could definitely do it the way you described, probably in 1 line like the Button Groups in [elemental-ui.com/buttons](http://elemental-ui.com/buttons).

**Comment by rtholmes** [comment]
The button group would work in this case with only two members, but if you had 8 people it might be too much. Individual buttons arranged vertically might be safer (and work better on mobile).
